nixos/doc/manual/release-notes/rl-2009.xml

```diff
@@ -474,6 +474,24 @@ systemd.services.nginx.serviceConfig.ReadWritePaths = [ "/var/www" ];
      <link xlink:href="https://github.com/NixOS/nixpkgs/issues/89205">#89205</link>.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     <varname>services.postfix.sslCACert</varname> was replaced by <varname>services.postfix.tlsTrustedAuthorities</varname>
+     which now defaults to the Nixpkgs certificate authorities (<varname>pkgs.cacert</varname>).
+    </para>
+    <para>
+     This enables postfix to validate TLS certificates for outgoing smtp connections by default. This should not have any
+     implications since most servers are running in opportunistic TLS mode which allows untrusted TLS connections to smtp servers.
+
+     For more details information to enforce only trusted TLS connections see
+     <link xlink:href="http://www.postfix.org/postconf.5.html#smtp_tls_security_level">smtp_tls_security_level</link>.
+    </para>
+    <para>
+     <varname>services.postfix.tlsTrustedAuthorities</varname> won't configure <literal>smtpd_tls_CAFile</literal> by default anymore.
+     This should not have any impact since <literal>smtpd_tls_CAFile</literal> is only relevant for servers with
+     <literal>smtpd_tls_ask_ccert = yes</literal> which needs to enabled explicitly by config.
```

Comment:
```suggestion
     This should not have any impact since <literal>smtpd_tls_CAFile</literal> is only relevant when
     <literal>smtpd_tls_ask_ccert = yes</literal>: the  <literal>smtpd_tls_ask_ccert</literal> configuration variable is not set by our Postfix configuration and then defaults to <literal>no</literal>.
```